.github/workflows/coordinator-test.yml

```diff
@@ -4,6 +4,53 @@ on:
   workflow_dispatch:
 
 jobs:
+
+  build:
+    name: Unit tests
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+      - uses: actions/setup-java@v3
+        name: Setup Java JDK
+        with:
+          distribution: 'temurin'
+          java-version: 8.0.332
```

Comment:
We can use this, but latest seems to be 362. fwtw.

Reply:
I think I should move this to just be `8` so we always get latest.